### ISSUE TITLE
[Performance] Leanplum metric initialization is very, very slow. 

### ISSUE THREAD

**Issue by hawkinsw** [mozilla-mobile/fenix#6679]
This will track a solution to the problem that initialization of Leanplum is very, very slow and blocks startup. After the resolution of #6196, initialization of this component accounts for a very large part of the app's startup time.

**Comment by hawkinsw** [comment]
See #6703

**Comment by mcomella** [comment]
@hawkinsw I noticed this was in the "Waiting" column. Is this still blocked? If not, what are the remaining action items for this issue?

**Comment by hawkinsw** [comment]
Blocked on Jonathan's work

**Comment by mcomella** [comment]
@hawkinsw What is Jonathan's work? The delayed push initialization https://github.com/mozilla-mobile/fenix/issues/6730?

**Comment by hawkinsw** [comment]
Yes

**Comment by mcomella** [comment]
@hawkinsw Can you elaborate on how that impacts Leanplum?

Are there additional action items we can take here? For example, I wonder if we should file investigations into:
- Calling the Leanplum SDK ourselves (as you suggested)
- Forking the Leanplum SDK to remove or postpone certain unnecessary calls

**Comment by hawkinsw** [comment]
I commented in #7448. I hope that answers your questions?

**Comment by mcomella** [comment]
Unassigning Will: not actively being worked on so I want to minimize confusion. This is still blocked on delaying push init.

**Comment by mcomella** [comment]
I think we're addressing Leanplum in its entirety in https://github.com/mozilla-mobile/fenix/issues/7448: closing.
